src/tests/ftest/util/ior_test_base.py

```diff
@@ -385,3 +387,33 @@ def verify_pool_size(self, original_pool_info, processes):
             self.fail(
                 "Pool Free Size did not match: actual={}, expected={}".format(
                     actual_pool_size, expected_pool_size))
+
+    def execute_cmd(self, cmd, fail_on_err=True, display_output=True):
+        """Execute cmd using general_utils.pcmd
+
+          Args:
+            cmd (str): String command to be executed
+            fail_on_err (bool): Boolean for whether to fail the test if command
+                                execution returns non zero return code.
+            display_output (bool): Boolean for whether to display output.
+        """
+        try:
+            # execute bash cmds
+            self.ret = pcmd(
```

Comment:
why self.ret ? Are we using it in any other context ?

Reply:
I need to use it at line 139 in io/dfuse_sparse_file.py. Plus it's good to store the return code in a variable which can be used by inheriting class.

Reply:
Done